### PR TITLE
Minor update to admin secrets panel (free antags for everyone)

### PR DESCRIPTION
**Pull request by Holoo-1**
## About The Pull Request
Remakes a button in admin secrets panel from everyone is traitor to everyone is admin chosen antag.

## Why It's Good For The Game
Slightly better buttons for admin to push

## Changelog
:cl:
admin: remade everyone is traitor into everyone is antag in secrets panel 
/:cl:

### CODE REVIEW

**Comment by Justice12354** [comment]
For administrative purposes, could you add Cultist to the possible antags, pretty please?

**Comment by Holoo-1** [comment]
Sure

**Comment by Justice12354** [comment]
i love you

File: code/modules/admin/verbs/secrets.dm [hunk 1]
```diff
@@ -457,24 +457,32 @@ GLOBAL_DATUM(everyone_a_traitor, /datum/everyone_is_a_traitor_controller)
...
 				tgui_alert(usr,"The game hasn't started yet!")
 				return
-			if(GLOB.everyone_a_traitor)
-				tgui_alert(usr, "The everyone is a traitor secret has already been triggered")
+			if(GLOB.everyone_an_antag)
+				var/are_we_antagstacking = tgui_alert(usr, "The everyone is antag secret has already been triggered. Do you want to stack antags???", "Are you sure about this?", list("Confirm", "Abort"))
+				if(are_we_antagstacking != "Confirm")
+					return
+
+			var/chosen_antag = tgui_input_list(usr, "Choose antag", "Chose antag", list("Traitor", "Changeling", "Heretic", "Ninja", "Wizad", "Nightmare"))
```
Comment (Watermelon914) [inline code/modules/admin/verbs/secrets.dm:471]: Wizad?
Make these defines

**Comment by Jacquerel** [comment]
if you make everyone on the station wizad does it teleport them all to the wizard station at the same time?

**Comment by Holoo-1** [comment]
Yea, everyone will be teleported

**Comment by Jacquerel** [comment]
that seems like a good moment for someone to simply fireball a single tile which contains every player on the server, however if you are turning everyone into a wizard I guess this is what you are expecting

**Comment by Holoo-1** [comment]
notes and things to fix (please help):

- [ ] latejoin antags always get implant uplink

- [ ] cultists may get a sacrifice targed which is also a cultist (teamkill)

- [ ] sometimes ling anag tgui does not show objectives in Intro tab. TP shows that objectives exist

https://github.com/tgstation/tgstation/assets/38228316/540db24b-3bbb-46fe-b5d5-4bea69b2f485

**Pull request merged**